Formula/fontforge.rb

```diff
@@ -101,6 +101,18 @@ def install
     end
   end
 
+  def caveats
+    <<-EOS.undent
```

Comment:
I wildly appreciate how anal I am, but heavily prefer `def caveats; <<-EOS.undent` with the caveats starting underneath that.

It's the most consistent style throughout formula and I'd generally like to start nudging non-conforming formulae towards it rather than allowing others to drift away. It also saves us from a pointless double indent where people indent two spaces for `<<-EOS.undent` and then indent a further two spaces to actually start the caveats on the line under `<<-EOS.undent`.